dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -918,6 +925,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 		hover: {
 			headlineByline: hoverHeadlineByline(format),
 			pagination: hoverPagination(format),
+			liveBlogStandfirst: hoverLiveblog(format),
```

Comment:
Maybe `standfirstLink` is a better name here? Including 'live blog' in the name makes it specific to blogs but we want to use `format` to make that choice.